WorldPainter/WPCore/src/main/java/org/pepsoft/worldpainter/exporting/WorldPainterChunkFactory.java

```diff
@@ -88,11 +89,14 @@ public ChunkCreationResult createChunk(Tile tile, int chunkX, int chunkZ) {
         final Chunk chunk = platformProvider.createChunk(platform, chunkX, chunkZ, maxHeight);
         result.chunk = chunk;
         final int minY = dimension.getMinHeight(), maxY = maxHeight - 1;
+
         final boolean biomesSupported2D = platform.capabilities.contains(BIOMES);
```

Comment:
The 2D and 3D biomes stuff is left over from copying this from old code which supported multiple formats. It's only still there because I'm still working on the Minecraft 1.18 support. It should be taken out altogether. This class will only ever encounter names biomes.